### ISSUE TITLE
error: ‘int64_t’ in namespace ‘std’ does not name a type

### ISSUE THREAD

**Issue by cellux** [hjson/hjson-cpp#40]
I tried to build the library on my Arch Linux box and got this error:

```
$ make
[ 20%] Building CXX object src/CMakeFiles/hjson.dir/hjson_decode.cpp.o
In file included from /home/rb/src/hjson-cpp/src/hjson_decode.cpp:1:
/home/rb/src/hjson-cpp/include/hjson/hjson.h:312:8: error: ‘int64_t’ in namespace ‘std’ does not name a type
  312 |   std::int64_t to_int64() const;
      |        ^~~~~~~
make[2]: *** [src/CMakeFiles/hjson.dir/build.make:76: src/CMakeFiles/hjson.dir/hjson_decode.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:98: src/CMakeFiles/hjson.dir/all] Error 2
make: *** [Makefile:91: all] Error 2
```

I think the issue is a missing `#include <cstdint>` from the top of `json.h`.

**Comment by trobro** [comment]
Thanks, will fix. I assume you are using gcc 13?

**Comment by cellux** [comment]
Indeed.

```
$ gcc --version
gcc (GCC) 13.1.1 20230429
```

**Comment by trobro** [comment]
Fixed in release 2.4
